src/WorldOfWarcraft/program.ts

```diff
@@ -993,15 +1000,33 @@ void calcBillboardMat(inout mat4 m) {
     m[2] = vec4(upVec, 0.0);
 }
 
+mat4 getBoneTransform(uint boneIndex) {
+    vec4 mx = texelFetch(u_TextureBoneMatrix, ivec2(1, boneIndex), 0);
```

Comment:
You need a `TEXTURE()` here to make it work on WebGPU:

```suggestion
    vec4 mx = texelFetch(TEXTURE(u_TextureBoneMatrix), ivec2(1, boneIndex), 0);
```